.env.example

```diff
@@ -12,3 +12,5 @@ GITHUB_TOKEN=<TOKEN FOR OAUTH INSTRUCTLAB MEMBER LOOKUP>
 TAXONOMY_REPO_OWNER=<GITHUB_ACCOUNT>
 TAXONOMY_REPO=<REPO_NAME>
```

Comment:
Are `TAXONOMY_REPO_OWNER` and `TAXONOMY_REPO` still in use after this PR?

Reply:
@toraponibm thanks for catching that. I had to switch the ENVs for be prefixed with `NEXT_PUBLIC_` for them to be accessible by the client side in NextJs.